data-default-class/data-default-class.cabal

```diff
@@ -15,4 +15,10 @@ source-repository head
 
 Library
   Build-Depends:     base >=2 && <5
+  if impl(ghc == 7.4.*)
```

Comment:
Semantically this means:

if this package is to be compiled with `ghc` (or `ghcjs` -- that's a long story)  *and* GHC's version happens to be `>= 7.4 && < 7.5`, *then* do...